README.md

```diff
@@ -239,25 +239,48 @@ Amplitude.getInstance().clearUserProperties();
 
 # Tracking Revenue #
 
-To track revenue from a user, call `logRevenue()` each time a user generates revenue. For example:
+The preferred method of tracking revenue for a user now is to use `logRevenueV2()` in conjunction with the provided `Revenue` interface. `Revenue` instances will store each revenue transaction and allow you to define several special revenue properties (such as revenueType, productId, etc) that are used in Amplitude dashboard's Revenue tab. You can now also add event properties to the revenue event, via the revenueProperties field. These `Revenue` instance objects are then passed into `logRevenueV2` to send as revenue events to Amplitude servers. This allows us to automatically display data relevant to revenue on the Amplitude website, including average revenue per daily active user (ARPDAU), 1, 7, 14, 30, 60, and 90 day revenue, lifetime value (LTV) estimates, and revenue by advertising campaign cohort and daily/weekly/monthly cohorts.
```

Comment:
I don't know what our general philosophy is on this, but it feels like this is more detail than necessary for SDK documentation. Specifically, when we change dashboard functionality I don't want to have to remember to update SDK docs as well. Can we just link to the actual revenue docs?